src/libs/ck-libs/io/Makefile

```diff
@@ -18,7 +18,6 @@ $(LIB): $(LIBOBJ)
 
 headers: $(HEADERS)
 	cp $(HEADERS) $(CDIR)/include/
-	touch headers
```

Comment:
What's the reason for this change?

Reply:
It didn't appear it was actually used for anything useful when we looked - we can add it back if needed?

Reply:
It probably creates the `headers` file so that the target "headers" two lines above isn't executed multiple times. I'd suggest adding it back, just in case.

Reply:
Resolved!